Libplanet.Explorer/GraphTypes/ActionType.cs

```diff
@@ -13,13 +17,41 @@ public ActionType()
                 "Arguments",
                 resolve: ctx =>
                 {
+                    object ConvertValue(IValue value)
+                    {
+                        switch (value)
+                        {
+                            case Text text:
+                            case Integer integer:
+                            case Bencodex.Types.Boolean boolean:
+                            case Null n:
+                                return value;
+                            case Binary binary:
+                                return binary.ToString();
+                            case List list:
+                                return list.Select(ConvertValue).ToList();
+                            case Dictionary dictionary:
+                                var d = new Dictionary<object, object>();
+                                foreach (var pair in dictionary)
+                                {
+                                    d.Add(ConvertValue(pair.Key), ConvertValue(pair.Value));
+                                }
+
+                                return d;
```

Comment:
This can be onelined using `.ToDictionary()` method:

```csharp
dictionary.ToDictionary(kv => ConvertValue(kv.Key), kv => ConvertValue(kv.Value));
```